### DISCUSSION TITLE
[Ineligible for community support] Cannot explain why messages with TTL are discarded after multiple redeliveries

### DISCUSSION THREAD

**Discussion by nakurienota** [rabbitmq/rabbitmq-server#10556] [Other]
Hello,
We have cluster RabbitMQ 3.11.2 Erlang 25.1.1, quorum queries.
Publisher on java uses confirms and returns. On the way to consumer right after rabbit:5672 we have tcpdump.
Sometimes publisher got all confirm/ no returns , but in tcpdump we don't see this packets. It is not system problem , it happens sometimes, it is not depends on load, could be 1 time a week, could be huge lag for 5-10 minutes, could be low time lag, they lose partially, not full, and we don't have connection problems with consumer. 
Part of messages, which have ttl, removes, but part of our messages are without ttl, we should guarantee deliever and that is behaviour we see for them. 
Publisher sent message -> got confirm -> message inside query, unacked state, but tcpdump doesn't contain these message -> timeout -> message redelivered ->  message inside query , x-delivery-count increased and this may happen several time before consumer gets message.
Could someone help or describe what happening?

**Comment by michaelklishin** [comment]
We cannot suggest anything without an executable way to reproduce. We do not guess in this community, in particular with features such as message TTL or multiple (expected or not) redeliveries to consumers are involved.

It is **your** responsibility to demonstrate a behavior.

**Comment by michaelklishin** [comment]
RabbitMQ 3.11 is [out of community support](https://rabbitmq.com/versions.html). If you have a support subscription with VMware, please put together an executable way for us to reproduce, then file a support ticket.
